drafts/SC1-4-1-link-in-text-style.md

```diff
@@ -0,0 +1,172 @@
+---
+rule_id: SC1-4-1-link-in-text-style
+name: Inline links are distinguishable
+test_mode: automatic
+
+criteria:
+- 1.4.1 # Use of Color (Level A)
+
+authors:
+- Wilco Fiers
+---
+
+## Description
+
+This rule checks that links that are embedded in a block of text can be distinguished in a way that does not rely exclusively on color. If this is the case, the link can only pass if the color contrast is sufficient, and additional cues are given when the link receives focus or hover.
+
+## Background
+
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/F73.html)
+- [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G183)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/C15:)
```

Comment:
http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G183
should be 
https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/G183
(assuming that 2014 is still our baseline?)